src/patternfly/components/Progress/progress.scss

```diff
@@ -42,18 +43,23 @@
     --pf-c-progress__bar--Height: var(--pf-c-progress__bar--Height--sm); // set height to the small height
   }
 
+  // No Description or Status Modifier
+  &.pf-m-nodescription {
+    grid-gap: 0;
+  }
+
   // an inside-style progress component shows the measure inside the indicator
   &.pf-m-inside {
     .pf-c-progress__indicator {
       display: flex;
       align-items: center;
       justify-content: center;
-      min-width: var(--pf-c-progress__indicator--inside--MinWidth);
+      min-width: var(--pf-c-progress__indicator--m-inside--MinWidth);
     }
 
     .pf-c-progress__measure {
-      font-size: var(--pf-c-progress__measure--inside--FontSize);
-      color: var(--pf-c-progress__measure--inside--Color);
+      font-size: var(--pf-c-progress__measure--m-inside--FontSize);
+      color: var(--pf-c-progress__measure--m-inside--Color);
```

Comment:
change this and the var above it to `--pf-c-progress--m-inside__measure--Color/FontSize`?